src/Management/src/EndpointCore/CloudFoundry/CloudFoundrySecurityMiddleware.cs

```diff
@@ -126,24 +113,6 @@ private IEndpointOptions FindTargetEndpoint(PathString path)
         {
             List<IEndpointOptions> configEndpoints;
 
-            // Remove in 3.0
```

Comment:
@hananiel is this what is meant by "Remove in 3.0" or is it supposed to be replaced with something else?